python-sdk/src/astro/databases/mssql.py

```diff
@@ -392,6 +392,34 @@ def _build_merge_sql(
         }
         return statement, params
 
+    def openlineage_dataset_name(self, table: BaseTable) -> str:
+        """
+        Returns the open lineage dataset name as per
+        https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md
+        Example: schema_name.table_name
```

Comment:
this example does not look right, it should be `database.schema.table.name`?

Reply:
```suggestion
        Example: database.schema.table.name
```